src/hotspot/share/classfile/javaClasses.hpp

```diff
@@ -1111,17 +1111,17 @@ class jdk_internal_vm_Continuation: AllStatic {
  public:
   static void serialize_offsets(SerializeClosure* f) NOT_CDS_RETURN;
   // Accessors
-  static inline oop scope(oop ref);
-  static inline oop target(oop ref);
-  static inline oop parent(oop ref);
-  static inline oop yieldInfo(oop ref);
-  static inline void set_yieldInfo(oop ref, oop value);
-  static inline stackChunkOop tail(oop ref);
-  static inline void set_tail(oop ref, stackChunkOop value);
-  static inline bool on_local_stack(oop ref, address adr);
-  static inline bool done(oop ref);
-  static inline bool is_preempted(oop ref);
-  static inline void set_preempted(oop ref, bool value);
+  static inline oop scope(oop continuation);
+  static inline oop target(oop continuation);
+  static inline oop parent(oop continuation);
+  static inline oop yieldInfo(oop continuation);
+  static inline void set_yieldInfo(oop continuation, oop value);
+  static inline stackChunkOop tail(oop continuation);
+  static inline void set_tail(oop continuation, stackChunkOop value);
+  static inline bool on_local_stack(oop continuation, address adr);
+  static inline bool done(oop continuation);
+  static inline bool is_preempted(oop continuation);
+  static inline void set_preempted(oop continuation, bool value);
```

Comment:
Thank you.  I've been trying to make the code say "continuation" when it's referring to the oop, and cont_entry or entry when it's referring to the ContinuationEntry.  I think the variable name 'cont' should be mostly confined to continuation.cpp to refer to the ContinuationWrapper in that file.  But there are other places that it's leaked out.